google/google-how-to/gce/set-hostname-using-cloudinit.rst

```diff
@@ -1,20 +1,29 @@
 Set hostname of GCE instances
-=================================
+=============================
 
-The hostname of GCE instances can be set using multiple methods. Apart from setting it manually, one option is to use a set-up tool like cloud-init, while Google's preferred method is to use its DHCP service. Both these options are considered here.
+The hostname of GCE instances can be set using multiple methods. Google's preferred method is to use its DHCP service, which requires you to choose a fully qualified DNS name (FQDN), e.g. something like test123.test.com. If you don't want an FQDN or if you want a specific format for consistent naming across clouds, you can use a set-up tool like cloud-init to set your hostname.
```

Comment:
"The hostname of GCE instances can be set using multiple methods. Google's preferred method is to use its DHCP service, which requires you to choose a fully qualified DNS name (FQDN), e.g. something like test123.test.com. If you don't want an FQDN or if you want to use a consistent method for assigning hostnames across clouds, you can use a set-up tool like cloud-init to set your hostname."

Reply:
Done.